test/test_torch.py

```diff
@@ -2363,39 +2363,263 @@ def _test_rpow(self, cast):
     def test_rpow(self):
         self._test_rpow(self, lambda x: x)
 
+    @staticmethod
+    def _check_against_np(self, tensor, exp):
+        tensor_np = tensor.cpu().numpy()
+        exp_np = exp if isinstance(exp, int) else exp.cpu().numpy()
+        expected = torch.LongTensor(tensor_np ** exp_np).type_as(tensor)
```

Comment:
LongTensor is a legacy constructor. We should update this to 
```
torch.tensor(tensor_np ** exp_np, dtype=tensor.dtype)
```

Reply:
addressed